programming-basics/palindromo/README.md

```diff
@@ -0,0 +1,26 @@
+# Palíndromo
+
+Neste exercício é necessário verificar se ao comparar duas `strings` estas são uma [palíndromo](https://pt.wikipedia.org/wiki/Pal%C3%ADndromo) ou seja, se estas `strings` possuem a propriedade de poder ser lida tanto da direita para a esquerda como da esquerda para a direita.
+
+Logo a sua função deverá retornar:
+
+- `true` quando o par de  `strings` for um **palíndromo**.
+- `false` quando o par de  `strings` não for um **palíndromo**.
+
+Vamos exemplificar usando alguns casos de entrada
+
+```javascript
+palindromo('ovo', 'ovo') === true
```

Comment:
I think palindromes doesn't work that way.... 

The function should receive **only one word** and check if this word is a palindrome 
```js
const isPalindrome = word => word === word.reverse()
```

Reply:
This is not an implementation, is just a high level abstraction